extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -48,11 +48,6 @@ public static function onBeforePageDisplay( OutputPage $out, Skin $skin ) {
 	public static function onOasisSkinAssetGroups( &$jsAssets ) {
 		global $wgWikiaEnvironment, $wgNoExternals, $wgRecirculationTestGroup, $wgCityId;
 
-		// We only want to track this on production
-		if ( ( $wgWikiaEnvironment === WIKIA_ENV_PROD ) && empty( $wgNoExternals ) && !empty( $wgRecirculationTestGroup ) ) {
-			$jsAssets[] = 'recirculation_trackers_js';
-		}
-
 		if ( self::isCorrectPageType() ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)